examples/jupyter/Modin_Taxi.ipynb

```diff
@@ -13,7 +13,7 @@
    "source": [
     "# To run this notebook as done in the README GIFs, you must first locally download the 2015 NYC Taxi Trip Data.\n",
     "import urllib.request\n",
-    "url_path = \"https://modin-datasets.s3.amazonaws.com/testing/yellow_tripdata_2015-01.csv\"\n",
+    "url_path = \"https://modin-datasets.intel.com/testing/yellow_tripdata_2015-01.csv\"\n",
```

Comment:
@leshikus Modin users will no longer be able to run these examples locally? (because of credentials)

Reply:
HTTPS access will work without credentials